backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/eid/EIdPosition.java

```diff
@@ -0,0 +1,35 @@
+package de.bund.digitalservice.ris.norms.domain.entity.eid;
+
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import org.w3c.dom.Element;
+
+/**
+ * The position information of a {@link EIdPart}.
+ */
+public sealed interface EIdPosition
+  permits OrdinalEIdPosition, ZaehlbezeichnungsbasierteEIdPosition {
+  /**
+   * Find the position information that should be used for the last part of the eId of the element.
+   * @param element the element whose position information should be determined
+   * @param eIdPartType the {@link EIdPartType} of the element
+   * @return the position information for the last part of the eId for the element
+   */
+  static EIdPosition findEIdPosition(Element element, EIdPartType eIdPartType) {
+    if (eIdPartType.equals(EIdPartType.ART) || eIdPartType.equals(EIdPartType.ABS)) {
```

Comment:
Shouldn't we check also here if the ART or ABS are actually not within a quotedStructure?

Reply:
They can use the `z` prefix in a quoted structure as well, there is no restriction on that. The difference within a quoted structure is that akn:article is using the parent elements eId as a prefix (like every other element) and not getting any special treatment for that.

Reply:
oh wait i thought the new z-prefixing was specifically only for articles within quotedStructure. It doesn't feel right that the standard is not restricting what prefixing is used when doing the ordinal, specially if the parents would have the n-prefix but then children possibly have the z-prefix?

Reply:
the articles (outside of `quotedStructure`) do not use the parent eId stuff at all anymore so they all have eIds like `art-z1`, `art-z3b` or (in case of `@refersTo="ordinale-zaehlung-eid"` `art-n4`). The idea is to make references to articles simpler by no longer needing the whole structure of the norm. 
Even in case of the ordinale-zaehlung (n-prefix) the articles do not include the parent part so if they are eg. in different akn:section elements they still are counted over the whole document.

There are two case in which a z-prefix part can follow one with a n-prefix: either when something is inside a quoted structure or when a paragraph is within an article with `@refersTo="ordinale-zaehlung-eid"`.